src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/X509Pal.PublicKey.cs

```diff
@@ -51,6 +56,50 @@ public AsymmetricAlgorithm DecodePublicKey(Oid oid, byte[] encodedKeyValue, byte
             }
         }
 
+        private static ECDsa DecodeECDsaPublicKey(byte[] encodedKeyValue)
+        {
+            if (encodedKeyValue.Length < 1 || (AsnTag)(encodedKeyValue[0]) != AsnTag.OCTET_STRING)
+                throw new CryptographicException(SR.InvalidPublicKeyInX509);
+
+            // After the OCTET_STRING tag, the encryptedKeyValue are X, followed by Y, in big-endian format.
+            BCRYPT_ECCKEY_BLOB_MAGIC magic;
+            switch (encodedKeyValue.Length - 1)
+            {
+                case 2 * ((256 + 7) / 8):
```

Comment:
(<bits> + 7)/8 == bit to byte length conversion.
2* since there are two octets (X and Y).

A named constant would obscure the key bit-length which needs to match up with the key bit-length named in the blob magic constant.